bluemira/builders/tools.py

```diff
@@ -85,7 +85,7 @@ def get_n_sectors(no_obj, degree=360):
 
 
 def circular_pattern_component(
-    component: bm_comp.Component,
+    component: Union[bm_comp.Component, List[bm_comp.Component]],
```

Comment:
This rearrangement means you can have multiple components all under the same sector patterned without ending up with an extra level